inst/unitTests/runit.sparseConversion.R

```diff
@@ -674,12 +674,19 @@ if (.runThisTest) {
         checkEquals(dgc, asSpMat(ind), msg="ind2dgC_9")
     } 
     if (suppressMessages(require(slam))) {
+        ## simple_triplet_matrix from package slam
         test.as.stm2dgc <- function() {
-            ## simple_triplet_matrix from package slam
             stm <- as.simple_triplet_matrix(diag(2))
             dgc <- as(as(stm, "matrix"), "dgCMatrix")
             checkEquals(dgc, asSpMat(stm), msg="stm2dgc")
         }
+        test.as.stm2stm <- function() {
+            if ("package:Matrix" %in% search())
+                uppressMessages(unloadNamespace("Matrix"))
```

Comment:
That is a type:  `suppressMessages()`.

I am not a fan of trying to unload packages. I don't think it works reliably.  What are you trying to do?

Reply:
Just to check that even without Matrix loaded, the conversion stm -> arma -> stm works well.
If unloading is a concern, we can do without it but we have to have a faith that it works without Matrix.

Reply:
That is how the unit tests work (eg on Travis):  if not explicitly loaded, it is not there. I'd remove the line with the typo and the preceding test.

Reply:
But it is explicitely loaded in runit.sparseConversion.R:34
```
.runThisTest <- suppressMessages(require(Matrix))
```

Reply:
That's the point. We _need_ Matrix for the existing conversions.

Reply:
Right. That's why I have unloaded it just for one test and reloaded it immediately after. But as I already said, it is OK for me to do without unload/reload. I have checked on my side that it worked without Matrix.